### PR TITLE
New cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Includes cal csvs for sensors returned from vendor refurb
Also includes 2 cruise-specific Papa 9 ADCPS-Ls on FLMA/B-9 (refer to 3408-00201 cal csv doc)

### CODE REVIEW

File: calibration/ADCPSL/CGINS-ADCPSL-20346__20220511.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+20346,CC_scale_factor1,0.45,constant; GP03FLMA-00009
+20346,CC_scale_factor2,0.45,constant
+20346,CC_scale_factor3,0.45,constant
+20346,CC_scale_factor4,0.45,constant
+20346,CC_bin_size,1000,in cm
+20346,CC_dist_first_bin,1647,in cm
+20346,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-20346__20220511.csv:8]: looks good

File: calibration/ADCPSL/CGINS-ADCPSL-24661__20220511.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+24661,CC_scale_factor1,0.45,constant; GP03FLMB-00009
+24661,CC_scale_factor2,0.45,constant
+24661,CC_scale_factor3,0.45,constant
+24661,CC_scale_factor4,0.45,constant
+24661,CC_bin_size,1000,in cm
+24661,CC_dist_first_bin,1647,in cm
+24661,CC_orientation,1,1 is upward looking; 0 is downward looking
```
Comment (cdobs) [inline calibration/ADCPSL/CGINS-ADCPSL-24661__20220511.csv:8]: looks good

File: calibration/CTDBPF/CGINS-CTDBPF-50062__20220224.csv [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+16-50062,CC_pa2,-6.927142e-12,
+16-50062,CC_ptca0,5.256217e+05,
+16-50062,CC_ptca1,3.915823e+01,
+16-50062,CC_ptca2,-7.828651e-01,
+16-50062,CC_ptcb0,2.480937e+01,
+16-50062,CC_ptcb1,6.750000e-04,
+16-50062,CC_ptcb2,0.000000e+00,
+16-50062,CC_ptempa0,-6.529720e+01,
+16-50062,CC_ptempa1,5.169256e+01,
+16-50062,CC_ptempa2,-2.847223e-01,
```
Comment (cdobs) [inline calibration/CTDBPF/CGINS-CTDBPF-50062__20220224.csv:23]: looks good

File: calibration/CTDBPF/CGINS-CTDBPF-50142__20220224.csv [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+16-50142,CC_pa2,-4.009924e-12,
+16-50142,CC_ptca0,5.248272e+05,
+16-50142,CC_ptca1,-5.552326e+00,
+16-50142,CC_ptca2,5.249233e-03,
+16-50142,CC_ptcb0,2.524618e+01,
+16-50142,CC_ptcb1,-5.764411e-04,
+16-50142,CC_ptcb2,0.000000e+00,
+16-50142,CC_ptempa0,-5.624179e+01,
+16-50142,CC_ptempa1,5.517100e+01,
+16-50142,CC_ptempa2,-8.500539e-01,
```
Comment (cdobs) [inline calibration/CTDBPF/CGINS-CTDBPF-50142__20220224.csv:23]: looks good

File: calibration/NUTNRB/CGINS-NUTNRB-01106__20220224.csv [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
...8, 273.28, 274.09, 274.89, 275.7, 276.5, 277.31, 278.11, 278.92, 279.72, 280.53, 281.33, 282.14, 282.94, 283.75, 284.56, 285.36, 286.17, 286.97, 287.78, 288.58, 289.39, 290.2, 291, 291.81, 292.61, 293.42, 294.22, 295.03, 295.84, 296.64, 297.45, 298.25, 299.06, 299.86, 300.67, 301.48, 302.28, 303.09, 303.89, 304.7, 305.5, 306.31, 307.12, 307.92, 308.73, 309.53, 310.34, 311.14, 311.95, 312.75, 313.56, 314.36, 315.17, 315.97, 316.78, 317.58, 318.39, 319.19, 320, 320.8, 321.61, 322.41, 323.22, 324.02, 324.83, 325.63, 326.44, 327.24, 328.04, 328.85, 329.65, 330.46, 331.26, 332.06, 332.87, 333.67, 334.48, 335.28, 336.08, 336.89, 337.69, 338.49, 339.3, 340.1, 340.9, 341.7, 342.51, 343.31, 344.11, 344.91, 345.72, 346.52, 347.32, 348.12, 348.92, 349.72, 350.53, 351.33, 352.13, 352.93, 353.73, 354.53, 355.33, 356.13, 356.93, 357.73, 358.53, 359.33, 360.13, 360.93, 361.73, 362.53, 363.33, 364.13, 364.93, 365.73, 366.53, 367.32, 368.12, 368.92, 369.72, 370.52, 371.31, 372.11, 372.91, 373.71, 374.5, 375.3, 376.1, 376.89, 377.69, 378.48, 379.28, 380.08, 380.87, 381.67, 382.46, 383.26, 384.05, 384.84, 385.64, 386.43, 387.23, 388.02, 388.81, 389.61, 390.4, 391.19, 391.98, 392.78, 393.57, 394.36]",
```
Comment (cdobs) [inline calibration/NUTNRB/CGINS-NUTNRB-01106__20220224.csv:8]: spot checked - looks good

File: calibration/PHSENF/CGINS-PHSENF-P0080__20220401.csv [hunk 6]
```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes
+P0080,CC_ea434,17372.0, source_file PHSEN-F_SAMI-pH_SN_P0080_Validation_Files_2022-04-01.zip > SAMI_P0080_010422T_ValCert.pdf
+P0080,CC_ea578,94.1,
+P0080,CC_eb434,2284.0,
+P0080,CC_eb578,38676.0,
+P0080,CC_ind_off,0,
+P0080,CC_ind_slp,1,
+P0080,CC_psal,35,Constant
```
Comment (cdobs) [inline calibration/PHSENF/CGINS-PHSENF-P0080__20220401.csv:8]: looks good

**Review by cdobs — APPROVED** [review]
looks good

**Comment by snwhite** [comment]
[PYIV Instrument Cal CSVs [DT-158]](https://trello.com/c/I1InRCiO/158-pyiv-instrument-cal-csvs-dt-158)

File: calibration/ADCPSL/CGINS-ADCPSL-20346__20220511.csv (hunk 1 above)
Comment (snwhite) [reply calibration/ADCPSL/CGINS-ADCPSL-20346__20220511.csv:8]: Looks good.

File: calibration/ADCPSL/CGINS-ADCPSL-24661__20220511.csv (hunk 2 above)
Comment (snwhite) [reply calibration/ADCPSL/CGINS-ADCPSL-24661__20220511.csv:8]: Looks good.

File: calibration/CTDBPF/CGINS-CTDBPF-50062__20220224.csv (hunk 3 above)
Comment (snwhite) [reply calibration/CTDBPF/CGINS-CTDBPF-50062__20220224.csv:23]: Looks good.

File: calibration/CTDBPF/CGINS-CTDBPF-50142__20220224.csv (hunk 4 above)
Comment (snwhite) [reply calibration/CTDBPF/CGINS-CTDBPF-50142__20220224.csv:23]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01106__20220224.csv (hunk 5 above)
Comment (snwhite) [reply calibration/NUTNRB/CGINS-NUTNRB-01106__20220224.csv:8]: Looks good.

File: calibration/PHSENF/CGINS-PHSENF-P0080__20220401.csv (hunk 6 above)
Comment (snwhite) [reply calibration/PHSENF/CGINS-PHSENF-P0080__20220401.csv:8]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**